packages/adminrouter/extra/systemd/dcos-adminrouter.service

```diff
@@ -13,16 +13,14 @@ PrivateDevices=yes
 StandardOutput=journal
 StandardError=journal
 PermissionsStartOnly=True
-Environment=AUTH_TOKEN_VERIFICATION_KEY_FILE_PATH=/var/lib/dcos/dcos-oauth/auth-token-secret
+Environment=AUTH_TOKEN_VERIFICATION_KEY_FILE_PATH=/run/dcos/dcos-adminrouter/auth-token-verification-key
 Environment=AUTH_ERROR_PAGE_DIR_PATH=${PKG_PATH}/nginx/conf/errorpages
-EnvironmentFile=/opt/mesosphere/etc/adminrouter.env
 EnvironmentFile=/opt/mesosphere/etc/dcos-oauth.env
+EnvironmentFile=-/opt/mesosphere/etc/adminrouter.env
```

Comment:
Why is this one now optional?

Reply:
Great question. I thought: hey, I did this on purpose but I don't know 'why' anymore! So I went into my commit messages where I wrote:
```
- Make inclusion of /opt/mesosphere/etc/adminrouter.env optional
  (also done downstream).
```
So, this is mainly for consistency. Secondly, this mechanism is a mechanism for opt-in / optional modification of Admin Router behavior.